hellosign_sdk/hsclient.py

```diff
@@ -794,61 +900,129 @@ def delete_template(self, template_id):
 
         return response
 
-    # def get_template_files(self, template_id):
-    def get_template_files(self, template_id, filename):
-        ''' Download a PDF copy of a template's original files
+    def get_template_files(self, template_id, path_or_file=None, file_type=None,
+            filename=None, get_url=False, get_data_uri=False):
+        ''' Downloads a copy of a template's original files
 
         Args:
 
-            template_id (str):  The id of the template to retrieve.
+            template_id (str): id of the template to download
+
+            path_or_file (str or file): A writable File-like object or a full path to save the PDF file to.
 
-            filename (str):     Filename to save the PDF file to. This should be a full path.
+            filename (str): [DEPRECATED] Filename to save the PDF file to. This should be a full path.
+
+            file_type (str): Type of file to return. Either "pdf" for a single merged document or
+            "zip" for a collection of individual documents. Defaults to "pdf" if not specified.
+
+            get_url (bool, optional): If True, returns a URL link to the file. Defaults to false.
+
+            get_data_uri (bool, optional): If True, returns the file as base64 encoded string. Defaults to False.
 
         Returns:
-            Returns a PDF file
+            Returns a PDF file, URL link to file, or base64 encoded file
 
         '''
-
-        url = self.TEMPLATE_GET_FILES_URL + template_id
         request = self._get_request()
+        url = self.TEMPLATE_GET_FILES_URL + template_id
+        if file_type:
+            url += '?file_type=%s' % file_type
+            return request.get_file(url, path_or_file or filename)
+
+        if get_url:
```

Comment:
So yeah, if this is the case that it's one or the other, and both modify the output format seems to want to be consolidated in a single property.